content/documentation/cli/cli-cmd-wallet-transaction-watch.mdx

```diff
@@ -15,6 +15,3 @@ Current Status... unconfirmed -> confirmed: 1m 30s
 Current Status... done after 2m 20s`}
 />
 
-## Next steps
-
-[Start mining $IRON](/use/get-started/mining).
```

Comment:
Removed the `next steps` in various places because it is redundant. 

if you hit the next button on the button of that documentation page it takes you to the next step which is also the page right underneath the current one in the left tab.